### PR TITLE
Fix BGM Volume after fadeout and add SE cancelling

### PR DESCRIPTION
**Pull request by Ghabry**

### CODE REVIEW

**Comment by carstene1ns** [comment]
Does this cancelling of fade effects also work with SDL_mixer / OpenAL?

**Comment by Ghabry** [comment]
SDL_Mixer removes the song completely after fadeout, which means SetVolume just won't work.
Need to check how RPG_RT behaves, if it restarts the song or continues.

File: src/scene_gamebrowser.cpp [hunk 1]
```diff
@@ -50,7 +50,6 @@ void Scene_GameBrowser::Continue() {
 #ifdef _WIN32
 	SetCurrentDirectory(L"..");
 #endif
-	Audio().BGM_Fade(800);
```
Comment (Ghabry) [inline src/scene_gamebrowser.cpp]: Removed this line because BGM_Stop is called few lines later.

**Comment by Ghabry** [comment]
Havn't tested the unloved OpenAL but it works now in the generic decoder and SDL2 Mixer.
Should work everywhere because it simply uses BGM_Play now.

**Pull request merged**